CoreFoundation/Base.subproj/CFPlatform.c

```diff
@@ -1325,21 +1336,35 @@ static void _CFThreadSpecificDestructor(void *ctx) {
 
 _CFThreadSpecificKey _CFThreadSpecificKeyCreate() {
     _CFThreadSpecificKey key;
+#if DEPLOYMENT_TARGET_WINDOWS
+    key = FlsAlloc(_CFThreadSpecificDestructor);
+#else
     pthread_key_create(&key, &_CFThreadSpecificDestructor);
+#endif
     return key;
 }
 
 CFTypeRef _Nullable _CFThreadSpecificGet(_CFThreadSpecificKey key) {
+#if DEPLOYMENT_TARGET_WINDOWS
+  return (CFTypeRef)FlsGetValue(key);
+#else
     return (CFTypeRef)pthread_getspecific(key);
+#endif
 }
 
 void _CFThreadSpecificSet(_CFThreadSpecificKey key, CFTypeRef _Nullable value) {
+#if DEPLOYMENT_TARGET_WINDOWS
+    if (value != NULL)
+        swift_retain((void *)value);
```

Comment:
Nit pick here - we always use curly braces or single-line style for if statements.